Add signalr args: 'service_mode' and 'allowed_origins' (#4716)
terraform-providers/terraform-provider-azurerm: website/docs/r/signalr_service.html.markdown

* `sku` - A `sku` block as documented below.

* `features` - (Optional) A `features` block as documented below.

> * `cors` - (Optional) A `cors` block as documented below.

Reviewer: _minor_ could we sort these alphabetically? e.g. `cors` -> `features` -> `sku`?

* `capacity` - (Required) Specifies the number of units associated with this SignalR service. Valid values are `1`, `2`, `5`, `10`, `20`, `50` and `100`.

---

A `features` block supports the following:

> * `flag` - (Required) A kind of feature. e.g. ServiceMode.

Reviewer: to match the other resources could we make this:

```suggestion
* `flag` - (Required) A kind of feature. At this time the only supported value is `ServiceMode`.
```

> * `value` - (Required) A value of a feature flag. e.g. Default, Serverless, or Classic.

Reviewer: to match the other resources could we make this:

```suggestion
* `value` - (Required) A value of a feature flag. Possible values are `Classic`, `Default` and `Serverless`.
```
